example.ini

```diff
@@ -0,0 +1,21 @@
+[app:main]
+factory = reddit_service_ads_tracking:make_wsgi_app
+
+metrics.namespace = reddit_service_ads_tracking
```

Comment:
should this change actually be made in puppet since this is just the example ini file?